workinggroups/storage.md

```diff
@@ -0,0 +1,32 @@
+# CNCF Storage WG Proposal
+
+## TOC Sponsor
+
+Ben Hindman
+
+## Objective
+
+Explore cloud native storage technology and concepts.
+
+## Goals and Expected Outcomes
+
+* Produce a landscape
+* Explore specifications for storage to propose to the CNCF
+* Bring recommendations to the TOC on storage projects in CNCF
```

Comment:
I would like to eliminate this as a goal. WGs are not gatekeepers for which projects the TOC considers.

Reply:
I didn't read this as implying that no one else could propose a storage project to the TOC.  Maybe could be re-worded as something like:

Evaluate storage project proposals, as well as potential outreach to projects that would be good CNCF projects, then provide data to the TOC on proposals

that's a little wordy, but I was trying to expand upon something discussed in a TOC meeting a while ago, where someone said that WG were supposed to help the TOC, providing more bandwidth with structure for volunteers to get involved, essentially expanding what the TOC can accomplish